dbgpt/model/proxy/llms/moonshot.py

```diff
@@ -99,3 +104,26 @@ def default_model(self) -> str:
         if not model:
             model = _MOONSHOT_DEFAULT_MODEL
         return model
+
```

Comment:
@Kain-90  Thanks for your PR， but the `generate_stream` function is unnecessary.  Please remove this.

Reply:
I'm sorry. You're right.